holoviews/core/data/pandas.py

```diff
@@ -16,7 +18,27 @@
 @lru_cache(maxsize=None)
 def deprecation_warning(msg):
```

Comment:
If this function can be reused elsewhere, then let's make it a general shared utility?

Reply:
I actually tried adding it to `holoviews.utils.__init__.py` but got `ImportError: cannot import name 'deprecation_warning' from partially initialized module 'holoviews.util' (most likely due to a circular import)`, which is why it stayed in the file (until now).